src/Microsoft.ML/CSharpApi.cs

```diff
@@ -15626,36 +15626,37 @@ public sealed class UniformRandomAutoMlEngine : AutoMlEngine
 
         public abstract class AutoMlStateBase : ComponentKind {}
 
-        public enum AutoInferenceAutoMlMlStateArgumentsMetrics
+        public enum PipelineSweeperSupportedMetricsMetrics
```

Comment:
Isn't MetricsMetrics a bit odd sounding?

Reply:
This is the auto generated file (the CSharpAPI translation layer) .  

The name here results from concatenating the following from PipelineSweeperSupportedMetrics.cs

<snip>
    public sealed class PipelineSweeperSupportedMetrics
    {
        public enum Metrics
        { 
</snip>

The name used for the class / enum look fine to me. Any alternate suggestions ?

---
In reply to: [203265071](https://github.com/dotnet/machinelearning/pull/539#discussion_r203265071) [](ancestors = 203265071)